### ISSUE TITLE
`livereload_script` should use "protocol-relative URL"

### ISSUE THREAD

**Issue by nulty** [tjwalch/django-livereload-server#66]
# Description

https://github.com/tjwalch/django-livereload-server/blob/95f9cf0f6bf4443a018afdfbdfcaff14cbda3f95/livereload/templatetags/livereload_tags.py#L9-L12

When this loads in my browser, the `src` populated with a relative URL path. The browser interprets this with the expectation that the path should be appended to the pages current URL.

The result is `http://localhost:8000/127.0.0.1:35729/livereload.js` which isn't going to work.

## Proposal
By adding the relative protocol syntax, the browser can interpret the script `src` as a complete URI with an empty scheme. The scheme is inherited from the pages scheme.

```
 """<script src="//{}:{}/livereload.js"></script>""", 
```
## Links

[Protocol Relative URL](https://en.wikipedia.org/wiki/URL#prurl)

**Comment by nulty** [comment]
This is actually addressed in https://github.com/tjwalch/django-livereload-server/pull/61

**Comment by kbuilds** [comment]
@nulty I don't think it was addressed. The change in #61 specifies the protocol to http. I like your proposal better.
